ignite/metrics/gan/fid.py

```diff
@@ -92,8 +74,8 @@ class FID(Metric):
     Args:
         num_features: number of features, must be defined if the parameter ``feature_extractor`` is also defined.
             Otherwise, default value is 2048.
-        feature_extractor: a callable for extracting the features from the input data. If neither num_features nor
-            feature_extractor are defined, default value is ``InceptionExtractor``.
+        feature_extractor: a callable for extracting the features from the input data. If neither ``num_features`` nor
+            ``feature_extractor`` are defined, default value is ``InceptionModel``.
```

Comment:
Better solution instead of exposing it to the user is to say that feature extractor will be based on InceptionV3 model and does not provide any link on the implementation